components/form/form-mixin.js

```diff
@@ -30,7 +27,9 @@ class Form extends LocalizeCoreElement(LitElement) {
 		this._validationCustoms = new Set();
 		this._errors = new Map();
 
-		this.addEventListener('d2l-form-errors-change', this._formErrorsChange);
+		this._form = document.createElement('form');
```

Comment:
This is created in the constructor so the setters in `d2l-form-native` can set properties on it. `firstUpdated` and `connectedCallback` are too late.